Dockerfile

```diff
@@ -92,12 +92,14 @@ RUN apk --update --no-cache add \
     openssl-dev \
     patch \
     pcre-dev \
-    php81-dev \
-    php81-pear \
+    php82-dev \
+    php82-pear \
     tar \
     tree \
     xz \
     zlib-dev
+	
+RUN gcc --version && g++ --version
```

Comment:
I want to avoid this as well, this adds extra cached layers and has not purpose for the image itself.

Reply:
Problem resolved.